content/docs/command-reference/repro.md

```diff
@@ -109,14 +111,21 @@ up-to-date and only execute the final stage.
   - `dvc repro -R pipelines/`: Directory path to explore recursively for
     `dvc.yaml` files
   - `dvc repro train-model`: Specific stage in `./dvc.yaml`
+  - `dvc repro model.pt`: Specific output file or directory listed in
+    `./dvc.yaml` (`outs` field)
   - `dvc repro modeling/dvc.yaml:prepare`: Stage in a specific `dvc.yaml` file
-  - `dvc repro --glob train-*`: Pattern to match groups of stages
+  - `dvc repro --glob 'model.??'`: Pattern to match groups of output filenames
+  - `dvc repro --glob-stages 'train-mode?'`: Pattern to match groups of stages
```

Comment:
```suggestion
  - `dvc repro --glob-stages cleaning/dvc.yaml:parse*`: Pattern to match groups
    of stages (by path and name)
```

Reply:
Question here: Is it intended approach to show commands with globs without putting them inside apostrophes (globs I mean). I don't know how does it work in other shells, but on my machine it will scan for filenames that match before arguments are passed to `dvc repro` command. Should it be like that?

Reply:
I guess it would need `'`s to avoid the shell's built-in globbing.